.github/workflows/build_base.yaml

```diff
@@ -0,0 +1,86 @@
+name: "Build leap"
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    outputs:
+      p:
+        description: "Discovered Build Platforms"
+        value: ${{ jobs.d.outputs.p }}
+
+permissions:
+  packages: write
```

Comment:
I think this should just be `read`, since below the `build-platforms` job has
```yaml
    permissions:
      packages: write
      contents: read
```
So I think as long as `build_base.yaml` is called with `packages: write`, which it seems to be:
https://github.com/AntelopeIO/leap/blob/4e19e3636237596aab66de2f6fceabcf89e0cc3f/.github/workflows/build.yaml#L34-L38
Having `packages: read` in this location should cause all jobs in this workflow other then `build-platform` be read, where `build-platform` is write.

(btw your ph_backward_compatibility.yaml already does this)
https://github.com/AntelopeIO/leap/blob/4e19e3636237596aab66de2f6fceabcf89e0cc3f/.github/workflows/ph_backward_compatibility.yaml#L6-L8

Reply:
Good catch. Slipped past when I was changing the others. I've corrected that now.